javascript/javascript2/week3/fp.js

```diff
@@ -0,0 +1,83 @@
+let input = document.getElementById('myinput');
```

Comment:
the game works pretty well! congrats,

but one thing really pained me, i couldn't press keys after i clicked start, i tried to find a way to get it to work, unfortunately :( but it would be worth looking into

Reply:
I will check again. You were not able to press S or L or any other key ?

Reply:
i had to click on the page before it would register, i found out that you can set the event on the window, that might work better 
```
window.addEventListener('keydown', ....
```

Reply:
Ok I have changed it now